frontend/app/src/components/StreamlitDialog/StreamlitDialog.test.tsx

```diff
@@ -55,11 +60,15 @@ describe("StreamlitDialog", () => {
       </Fragment>
     )
 
-    expect(wrapper.find("StyledSecondaryButton")).toMatchSnapshot()
+    const baseButtonSecondary = await screen.findByTestId(
+      "baseButton-secondary"
+    )
+    expect(baseButtonSecondary).toHaveFocus()
```

Comment:
Think this was meant for the `renders clear cache dialog and focuses clear cache button` test above 😄

Reply:
oops thanks 😅

Reply:
Done!